### ISSUE TITLE
running `pip install -e .` fails on Python 3.4

### ISSUE THREAD

**Issue by anthrotype** [DRMacIver/hypothesis#124]
hello,

I tried to to install hypothesis using pip "editable" mode -- the equivalent of setuptools "develop" -- but I'm getting the error below. However, if I do `python setup.py develop` directly it works.
BTW, thanks @DRMacIver for yesterday's talk at TLPG, it was very interesting!
Cheers,

Cosimo
 
```
$ pip install -e .
Obtaining file:///Users/cosimolupo/Documents/Github/hypothesis
Installing collected packages: hypothesis
  Running setup.py develop for hypothesis
    Complete output from command /usr/local/var/pyenv/versions/hypothesis/bin/python3.4 -c "import setuptools, tokenize; __file__='/Users/cosimolupo/Documents/Github/hypothesis/setup.py'; exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" develop --no-deps:
    running develop
    error: ("Can't get a consistent path to setup script from installation directory", '/', '/Users/cosimolupo/Documents/Github/hypothesis')
    
    ----------------------------------------
Command "/usr/local/var/pyenv/versions/hypothesis/bin/python3.4 -c "import setuptools, tokenize; __file__='/Users/cosimolupo/Documents/Github/hypothesis/setup.py'; exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" develop --no-deps" failed with error code 1 in /Users/cosimolupo/Documents/Github/hypothesis
```

**Comment by DRMacIver** [comment]
Yeah, someone else reported this recently. I need to look into it but haven't yet. FWIW, setup.py develop works fine as a workaround until I fix this.

**Comment by DRMacIver** [comment]
This should be fixed as of 4144123074a072b421b0f7c19726a617fa516fd0

**Comment by anthrotype** [comment]
thanks!
